src/config/rendering.ts

```diff
@@ -118,6 +132,11 @@ const envConfig: Record<Mode, Keys<RenderingConfig>> = {
     verboseLogging: 'GF_PLUGIN_RENDERING_VERBOSE_LOGGING',
     dumpio: 'GF_PLUGIN_RENDERING_DUMPIO',
     timingMetrics: 'GF_PLUGIN_RENDERING_TIMING_METRICS',
+    tracing: {
+      url: 'GF_PLUGIN_RENDERING_TRACING_URL',
+      serviceName: 'GF_PLUGIN_RENDERING_TRACING_SERVICE_NAME'
```

Comment:
This is not used by the `populateRenderingConfigFromEnv` function. This should either be used or removed. I think we can remove it as it's less likely to be an issue for people using the image renderer in plugin mode (usually, this means no Kubernetes setup where you already gave a name at your services). And we can always add it later if requested.